src/cdef.rs

```diff
@@ -20,11 +21,16 @@ use crate::src::cpu::{rav1d_get_cpu_flags, CpuFlags};
 #[cfg(feature = "asm")]
 use crate::include::common::bitdepth::BPC;
 
-pub type CdefEdgeFlags = c_uint;
-pub const CDEF_HAVE_BOTTOM: CdefEdgeFlags = 8;
-pub const CDEF_HAVE_TOP: CdefEdgeFlags = 4;
-pub const CDEF_HAVE_RIGHT: CdefEdgeFlags = 2;
-pub const CDEF_HAVE_LEFT: CdefEdgeFlags = 1;
+bitflags! {
+    #[repr(transparent)]
+    #[derive(Clone, Copy)]
+    pub struct CdefEdgeFlags: u32 {
+        const CDEF_HAVE_BOTTOM = 1 << 3;
+        const CDEF_HAVE_TOP = 1 << 2;
+        const CDEF_HAVE_RIGHT = 1 << 1;
+        const CDEF_HAVE_LEFT = 1 << 0;
```

Comment:
`c2rust transpile` also puts them backwards for some reason; could you flip them around?  So `1 << 0` at the top.